### PR TITLE
Svc -> Service

### PR DESCRIPTION
**Pull request by AudriusButkevicius**
Lets break all the PR's!

### CODE REVIEW

File: cmd/syncthing/main.go [hunk 1]
```diff
@@ -1045,17 +1036,13 @@ func ensureDir(dir string, mode os.FileMode) {
 		l.Fatalln(err)
 	}
 
-	if fi, err := os.Stat(dir); err == nil {
```
Comment (calmh) [inline cmd/syncthing/main.go]: Ap ap ap! Reverting stuff that shall not be reverted :D

File: lib/connections/connections.go [hunk 2]
```diff
@@ -470,9 +470,9 @@ func (s *connectionSvc) CommitConfiguration(from, to config.Configuration) bool
 	return true
 }
 
-// serviceFunc wraps a function to create a suture.Service without stop
+// connectionServiceFunc wraps a function to create a suture.Service without stop
 // functionality.
-type serviceFunc func()
+type connectionServiceFunc func()
```
Comment (calmh) [inline lib/connections/connections.go]: Not sure this rename is necessary, but OK

**Comment by calmh** [comment]
+ formatting of lib/relay/relay.go, otherwise mergeit

File: lib/connections/connections.go (hunk 2 above)
Comment (AudriusButkevicius) [inline lib/connections/connections.go]: This is a side-effect of Replace All, let me revert that.

**Comment by AudriusButkevicius** [comment]
Seems to be fine now.

**Pull request merged**